src/client/components/MessageList/MessageList.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import Message from './components/Messages/Message';
```

Comment:
Here is the path you need.

`./` means start in the current folder. Looks like you think it will start from the root of the project or something.

```suggestion
import Message from '../Messages/Message';
```

Reply:
Ok I got it. Now it is resolved. Do I close this request now? SO next time I iwll create a draft PR if changes not done